tools/bcr-mirror/README.md

```diff
@@ -0,0 +1,13 @@
+# BCR source archive mirror
+
+This directory contains a [Cloudflare Worker](https://developers.cloudflare.com/workers/) that mirrors source archives for modules in a Bazel registry to a [Cloudflare R2](https://developers.cloudflare.com/r2/) bucket.
+The bucket can then be used with Bazel's `--module_mirrors` flag (available as of Bazel 8.4.0).
```

Comment:
Which URL will be used?

Reply:
OK, ideally we can just use `bcr-mirror.bazel.build`, but we can update this later.